gateway/tests/api/test_job.py

```diff
@@ -83,6 +83,19 @@ def test_job_detail(self):
         self.assertEqual(jobs_response.data.get("status"), "SUCCEEDED")
         self.assertEqual(jobs_response.data.get("result"), '{"somekey":1}')
 
+    def test_job_provider_detail(self):
```

Comment:
@akihikokuroda , just to confirm, do we have a test that checks if the user is not a provider and either the author of the job returns a 404?

Reply:
Yes, it is `def test_not_authorized_job_detail(self)`